src/main/java/com/emc/object/s3/bean/BucketDeletionStatus.java

```diff
@@ -0,0 +1,155 @@
+package com.emc.object.s3.bean;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import java.util.ArrayList;
+import java.util.List;
+
+@XmlRootElement(name = "EmptyBucketStatus")
+public class BucketDeletionStatus {
+    private String status;
+
+    private String created;
+
+    private String lastUpdated;
+
+    private long deleted;
```

Comment:
Shall we keep the same as ECS to define as `Long entriesDeleted` ?
And also the rest of types to be class?